www/install/php/Update-2.8.0.php

```diff
@@ -54,7 +54,6 @@
         $query = 'INSERT INTO host (host_name, host_register) '
             . 'VALUES ("_Module_Meta", "2") ';
         $pearDB->query($query);
```

Comment:
needed to get id of the previous insert query
```suggestion
        $pearDB->query($query);
        $res = $pearDB->query($queryHost);
```